eng/pipelines/coreclr/templates/crossgen-comparison-job.yml

```diff
@@ -116,13 +116,15 @@ jobs:
       - script: |
           mkdir $(workItemDirectory)\log\$(crossFlavor)
         displayName: Create directories
+        
 
     # Create baseline output on the host (x64) machine
     - task: PythonScript@0
       displayName: Create cross-platform crossgen baseline
       inputs:
         scriptSource: 'filePath'
         scriptPath: $(coreClrRepoRoot)/tests/scripts/crossgen_comparison.py
+        pythonInterpreter: /usr/bin/python3
```

Comment:
Do we need to add `python3` to our linux requirements markdown file?

Reply:
We should be one python3, seeing that python2 is eol. We have not exactly mandated it yet.